hadoop-tools/hadoop-sls/src/test/java/org/apache/hadoop/yarn/sls/TestSLSDagAMSimulator.java

```diff
@@ -28,9 +28,13 @@
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameters;
 
+import static org.junit.Assume.assumeNotNull;
```

Comment:
if you don't factor out the common assert, can you put the static import at the bottom of all other imports. thanks